Userland/Utilities/ntpquery.cpp

```diff
@@ -278,9 +286,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         outln("Transmit timestamp:    {:#016x} ({})", transmit_timestamp, format_ntp_timestamp(transmit_timestamp).characters());
         outln("Destination timestamp: {:#016x} ({})", destination_timestamp, format_ntp_timestamp(destination_timestamp).characters());
 
+#ifdef SO_TIMESTAMP
```

Comment:
Is it worth messing with this program, or should we just not build it at all `if (HURD)` in the CMakeLists.txt?

Reply:
Not that we need it on the Hurd (I only really care about Ladybird anyway), but also it works fine here, it just cannot report kernelspace-to-userspace latency. So, up to you.

Reply:
eh. we can keep it. It's just a bit more ifdef than I'd prefer.